python_scripts/shellies_discovery.py

```diff
@@ -1401,6 +1409,22 @@
     KEY_NAME: "Firmware",
     KEY_STATE_TOPIC: TOPIC_INFO,
 }
+OPTIONS_SENSOR_WINDOW_OPEN = {
+    KEY_ENABLED_BY_DEFAULT: False,
+    KEY_ENTITY_CATEGORY: ENTITY_CATEGORY_DIAGNOSTIC,
+    KEY_DEVICE_CLASS: DEVICE_CLASS_WINDOW,
+    KEY_NAME: "Window open",
+    KEY_STATE_TOPIC: TOPIC_INFO,
+    KEY_VALUE_TEMPLATE: TPL_WINDOW_OPEN,
+}
+
+OPTIONS_SENSOR_OPEN_WINDOW_REPORT = {
+    KEY_ENABLED_BY_DEFAULT: False,
+    KEY_ENTITY_CATEGORY: ENTITY_CATEGORY_DIAGNOSTIC,
+    KEY_NAME: "Open window report",
```

Comment:
IMO `Window status reporting` is a better name. Similarly, you should also change `OPTIONS_SENSOR_OPEN_WINDOW_REPORT` and `TPL_OPEN_WINDOW_REPORT`.

Reply:
Ok I took name from TRV settings, will change as suggested.